//
// PROJECT:         Aspia
// FILE:            category/category_video_adapter.proto
// LICENSE:         Mozilla Public License Version 2.0
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message VideoAdapter
{
    message Item
    {
        string description     = 1;
        string adapter_string  = 2;
        string bios_string     = 3;
        string chip_type       = 4;
        string dac_type        = 5;
        string driver_date     = 6;
        string driver_version  = 7;
        string driver_provider = 8;
        uint64 memory_size     = 9;
    }

    repeated Item item = 1;
}
